vcsclient/bitbucketserver.go

```diff
@@ -642,6 +644,58 @@ func (client *BitbucketServerClient) UploadCodeScanning(ctx context.Context, own
 	return "", errBitbucketCodeScanningNotSupported
 }
 
+type diffPayload struct {
+	Diffs []struct {
+		Source struct {
+			ToString string `mapstructure:"toString"`
+		} `mapstructure:"source"`
+		Destination struct {
+			ToString string `mapstructure:"toString"`
+		} `mapstructure:"destination"`
+	} `mapstructure:"diffs"`
+}
+
+func (client *BitbucketServerClient) GetModifiedFiles(ctx context.Context, owner, repository, refBefore, refAfter string) ([]string, error) {
+	err := validateParametersNotBlank(map[string]string{
+		"owner":      owner,
+		"repository": repository,
+		"refBefore":  refBefore,
+		"refAfter":   refAfter,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	bitbucketClient, err := client.buildBitbucketClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	params := map[string]interface{}{"contextLines": int32(0), "from": refBefore, "to": refAfter}
+	resp, err := bitbucketClient.StreamDiff_37(owner, repository, "", params)
```

Comment:
Is it intended to use StreamDiff_37 and not StreamDiff?

Reply:
Because for the StreamDiff you must provide `the source path for the file` as said in the [documentation](https://github.com/gfleury/go-bitbucket-v1/blob/711d7d5e805f/default_api.go#L10995)

Reply:
So it doesn't work for our case because it is not what we want to have.